### PR TITLE
Add support for template use

### PR DESCRIPTION
**Pull request by otavio**
This adds support to provide the template id when using the v3 API.

### CODE REVIEW

**Comment by otavio** [comment]
We extended the API with the items we need to support our use case. Please let me know if we need to adapt something so we can get this merged.

File: src/v3.rs [hunk 1]
```diff
@@ -25,11 +25,14 @@ pub struct V3Sender {
 pub struct SGMailV3 {
     from: Email,
     subject: String,
-    content: Vec<Content>,
     personalizations: Vec<Personalization>,
 
+    #[serde(skip_serializing_if = "Option::is_none")]
```
Comment (gsquire) [inline src/v3.rs]: Sorry this is a nitpick but could you add a space between fields here? The serde attributes make it a bit noisy.

File: src/v3.rs [hunk 2]
```diff
@@ -266,6 +300,11 @@ impl Attachment {
         self.content = BASE64.encode(c);
     }
 
+    /// The base64 body of the attachment.
+    pub fn set_base64_content(&mut self, c: &str) {
+        self.content = c.into();
```
Comment (gsquire) [inline src/v3.rs]: Can you use `String::from` here like above?

**Comment by otavio** [comment]
I did the requested changes, please take a look at it and let me know how it looks...

**Comment by gsquire** [comment]
Thanks for this change! I will have this published soon and will post the version here.

**Comment by otavio** [comment]
It worked! Please publish the 0.7.x branch so backport fixes can be applied while the 0.8 is under development.

**Pull request merged**